examples/extensions/add_prices.py

```diff
@@ -49,55 +40,43 @@ def main(client, customer_id, campaign_id):
     price_feed_item.language_code.value = 'en'
 
     # To create a price extension, at least three price offerings are needed.
+    price_extension_price_unit_enum = (
+        client.get_type('PriceExtensionPriceUnitEnum'))
     price_feed_item.price_offerings.extend([
-        create_price_offer(client,
-                    'Scrubs',
-                    'Body Scrub, Salt Scrub',
-                    60000000,  # 60 USD
-                    'USD',
-                    client.get_type('PriceExtensionPriceUnitEnum').PER_HOUR,
-                    'http://www.example.com/scrubs',
-                    'http://m.example.com/scrubs'),
-        create_price_offer(client,
-                    'Hair Cuts',
-                    'Once a month',
-                    75000000,  # 75 USD
-                    'USD',
-                    client.get_type('PriceExtensionPriceUnitEnum').PER_MONTH,
-                    'http://www.example.com/haircuts',
-                    'http://m.example.com/haircuts'),
-        create_price_offer(client, 'Skin Care Package',
-                    'Four times a month',
-                    250000000,  # 250 USD
-                    'USD',
-                    client.get_type('PriceExtensionPriceUnitEnum').PER_MONTH,
-                    'http://www.example.com/skincarepackage')
+        _create_price_offer(client, 'Scrubs', 'Body Scrub, Salt Scrub',
+            60000000,  # 60 USD
```

Comment:
There's a few ways to list a large number of arguments while following recommended Python style, but you're sort of mixing two approaches here. I'd recommend a look at [PEP8's description of indentation](https://www.python.org/dev/peps/pep-0008/#indentation) for guidance.

E.g. You could do:
```
        # If you really want to emphasize individual arguments.
        _create_price_offer(client,
                            'Scrubs',
                            'Body Scrub, Salt Scrub',
                            60000000,  # 60 USD
                            'USD',
                            price_extension_price_unit_enum.PER_HOUR,
                            'http://www.example.com/scrubs',
                            'http://m.example.com/scrubs'),
```

Or:

```
        _create_price_offer(client, 'Scrubs', 'Body Scrub, Salt Scrub',
                            60000000, 'USD',
                            price_extension_price_unit_enum.PER_HOUR,
                            'http://www.example.com/scrubs',
                            'http://m.example.com/scrubs'),
```

Or:

```
        _create_price_offer(
            client, 'Scrubs', 'Body Scrub, Salt Scrub', 60000000, 'USD',
            price_extension_price_unit_enum.PER_HOUR,
            'http://www.example.com/scrubs', 'http://m.example.com/scrubs'),
```

I generally prefer to go with the latter, but any of these would be fine, provided its consistent across the example.

Reply:
I'm doing the latter, but I left the comment re: USD in to be consistent with the golden example.

Reply:
It doesn't appear that you're doing the latter; you shouldn't have arguments on the first line in the case.

W.r.t. the golden CL's comment, maybe place it over the `_create_price_offer` call? I don't think precise placement is so critical that you'd need to do unconventional indentation to make it work.